app/com/gu/floodgate/AppComponents.scala

```diff
@@ -66,11 +68,16 @@ class AppComponents(context: Context) extends BuiltInComponentsFromContext(conte
   val runningJobController = new RunningJobApi(runningJobService)
   val jobHistoryController = new JobHistoryApi(jobHistoryService)
 
-  val appController = new Application(wsClient, configuration)
+  val clientId = configuration.get[String]("google.clientid")
```

Comment:
How do you feel about wrapping these 5 lines to avoid polluting the object with transient vals:
```
val authConfig = {
  val clientId = configuration.get[String]("google.clientid")
  val clientSecret = configuration.get[String]("google.clientsecret")
  val redirectUrl = configuration.get[String]("google.oauthcallback")
  val domain = "guardian.co.uk"
  GoogleAuthConfig(clientId, clientSecret, redirectUrl, domain)
}
```